ocaml/xapi/quicktest_vdi_copy.ml

```diff
@@ -51,6 +51,17 @@ let read_from_vdi ~session_id ~vdi f =
       Http.Get uri in
   http req (fun (_, fd) -> f fd)
 
+(* Simpler alternative to Xapi_vdi.wait_for_vbds_to_be_unplugged_and_destroyed,
```

Comment:
Should expose this function in the mli and call it directly?
I'm fine with the simple alternative though.

Reply:
I can't, because it's using `context` and not `rpc session_id` :/

Reply:
Lets keep the simple alternative then